node/src/components/binary_port.rs

```diff
@@ -821,13 +814,12 @@ where
     }
 }
 
-async fn client_loop<REv, const N: usize, R, W>(
-    mut server: JulietRpcServer<N, R, W>,
+async fn client_loop<REv>(
```

Comment:
This is a bit confusing to me, should this be named `handle_client_loop` instead? :)

Reply:
Renamed in e20b3ae07eee1b601cd624c9655a55adba9a86ab